### DISCUSSION TITLE
Testing results debugging

### DISCUSSION THREAD

**Discussion by thavlice** [course-go/homework#3] [Assignments]
Is there and easy way or perhaps an extension to VSCode-Goland which allows to see the results of Go tests (mainly the EXPECTED vs ACTUAL diffs) coloured or side by side nicely?

**Comment by stanislav-zeman** [comment]
The diffs you are talking about from the EGG homework are a result of the [testcript](https://pkg.go.dev/github.com/rogpeppe/go-internal/testscript) package not the testing framework itself.

Go itself does not output any diffs, you got to write those yourself.

As for the colours, I am unaware of any such tooling for tests. You could use some general colourizers that just parse the text output and colour it.

PS: Sorry for the late response. GitHub does not seem to support subscription to Discussions...
